client/modules/User/components/Notification.jsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from 'react';
+import Cookies from 'js-cookie';
+import { useDispatch } from 'react-redux';
+import { showToast, setToastText } from '../../IDE/actions/toast';
+
+function Notification() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    const notification = Cookies.get('p5-notification');
+    if (!notification) {
+      // show the toast
+      dispatch(showToast(30000));
+      const text = `There is a scheduled outage on Sunday, April 9 3AM - 5AM UTC.
```

Comment:
@catarak 
Is this on Saturday, April 9? Someone was having trouble with the date.

Reply:
This is on *Sunday* the 10th. Sorry, this is a confusion from converting EDT to UTC.